test/writer_test.js

```diff
@@ -30,4 +30,24 @@ describe('test writer module', function () {
     const writer = new Writer({}, app);
     return writer.write();
   });
+
+  it('getIndexContent shouldn\'t throws error', async function () {
```

Comment:
I don't understand this test. Shouldn't it be "getIndexContent() should throw an error if .... " in the description?